README.md

```diff
@@ -1,32 +1,79 @@
+# Overview
+TorchDriveEnv is a lightweight reinforcement learning benchmark for autonomous driving written entirely in python, that provides human like non-playable characters (NPCs). Installation is as easy as running:
+
+```
+pip install torchdriveenv[baselines]
+```
+
+The benchmark provides NPCs through the state of the art behavioral models provided through the [Inverted AI API](https://www.inverted.ai/home). While emphasis is on ease of use, TorchDriveEnv is fully customizable, with easy to define new scenarios or reward. TorchDriveEnv is built on the lightweight driving simulator [TorchDriveSim](https://github.com/inverted-ai/torchdrivesim/), and therefore comes with various kinematic models, agent types and traffic controls. 
+
+
+## Scenario overview
+TorchDriveEnv comes bundled with a predefined collection of diverse and challenging training and testing scenarios. Training scenarios were created using the InvertedAI API, while the testing scenarios were hand tailored for interesting challenges. An overview of the testing scenarios is given in the visualization below. Note that while the ego agent has fixed waypoints in these scenarios, its starting position is sampled uniformly between the first two waypoints, and NPCs are initialized dynamically using the InvertedAI API. This results in a large variation in the available scenarios. 
+
+<table>
+  <tr>
+    <td align="center"><img src="https://github.com/inverted-ai/torchdriveenv/assets/16724505/ab82ec1e-fe79-4721-a996-512162032894.png" alt="Three Way"><br>Three Way</td>
```

Comment:
The sizes of the images here are slightly different. Could you try:
`<img src="..." alt="..." width="100" height="100">` 
to resize the images here?

Reply:
Thank you! I was trying to figure out how to do this, but I was not able to figure this out. My concern is that if I set it to 100, it will always be a fixed size, which will look strange at large or small resolutions. Do you know any way to set it to the size of the table cell?